src/test/java/cloud/eppo/helpers/BanditTestCase.java

```diff
@@ -0,0 +1,36 @@
+package cloud.eppo.helpers;
+
+import java.util.List;
+
+public class BanditTestCase {
+  private final String flag;
+  private final String defaultValue;
+  private final List<SubjectBanditAssignment> subjects;
+  private String fileName;
```

Comment:
Set later (after deserialization) to make helpful test case "names" that are displayed in IDE/terminal